File: btmanager/src/main/java/karrel/com/btconnector/chatmanager/BluetoothChatListener.java
package karrel.com.btconnector.chatmanager;

/**
 * Created by Rell on 2018. 1. 29..
 */

public interface BluetoothChatListener {
    void onConnectedSuccess(String deviceName);

    void onConnecting(String deviceName);

    void onInitalized(String deviceName);

    void onMessageSend(byte[] writeBuf);

    void onReadMessage(byte[] readBuf);

    void onStartConnect(String deviceName);

    void onConnectedFail(String name);
}

File: btmanager/src/main/java/karrel/com/btconnector/chatmanager/BluetoothChatListener.kt
package karrel.com.btconnector.chatmanager

/**
 * Created by Rell on 2018. 1. 29..
 */

interface BluetoothChatListener {
    fun onConnected(deviceName: String?)

    fun onConnecting(deviceName: String?)

    fun onListening(deviceName: String?)

    fun onSendMessage(writeBuf: ByteArray?)

    fun onReadMessage(readBuf: ByteArray?)

    fun onStartConnect(deviceName: String?)

    fun onDoingNothing(deviceName: String?)

    fun onFailedConnect(deviceName: String?)

    fun onLostedConnect(deviceName: String?)
}
